lib/Doctrine/ORM/Cache/CacheConfiguration.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+declare(strict_types = 1);
```

Comment:
Just did a quick replace in PhpStorm. 😂 `^<?php` -> `<?php\n\ndeclare(strict_types = 1);\n`. Will adjust accordingly if this PR should land eventually.